samples/projects/awscdk-typescript/bin/bootstrap.ts

```diff
@@ -13,7 +14,7 @@ const stack = cdf.awscdk.initProjectStack(
   config as PackageInfraConfig, 
   new PackagePlanner(), 
   [
-    new CustomModuleOne(),
-    new CustomModuleAnother(),
+    // new CustomModuleOne(),
+    // new CustomModuleAnother(),
```

Comment:
Why are we excluding the modules?

Reply:
good catch. a trouble shooting step I forgot to revert.